website/content/api-docs/secret/databases/index.mdx

```diff
@@ -315,6 +315,27 @@ This endpoint creates or updates a role definition.
   functionality. See the plugin's API page for more information on support and
   formatting for this parameter.
 
+- `credential_type` `(string: "password")` – Specifies the type of credential that
+  will be generated for the role. Options include: `password`, `rsa_private_key`.
+  See the plugin's API page for credential types supported by individual databases.
+
+- `credential_config` `(map<string|string>: <optional>)` – Specifies the configuration
+  for the given `credential_type`.
+
+  The following options are available for each `credential_type` value:
+
+  - `password`
+    - `password_policy` `(string: <optional>)` - The [policy](/docs/concepts/password-policies)
+      used for password generation. If not provided, defaults to the password policy of the
+      database [configuration](/api-docs/secret/databases#password_policy).
+
+  - `rsa_private_key`
+    - `key_bits` `(int: 2048)` - The bit size of the RSA key to generate. Options include:
+      `2048`, `3072`, `4096`.
+    - `format` `(string: "pkcs8")` - The output format of the generated private key
+      credential. The private key will be returned from the API in PEM encoding. Options
+      include: `pkcs8`.
+
```

Comment:
This markdown is repeated for dynamic and static roles. Does anyone know how to have a single copy of this content and use references in markdown? 🤔

Reply:
Yep! You can use [partials](https://github.com/hashicorp/vault/tree/main/website#markdown-enhancements).

Reply:
Oh cool, thanks! Will give that a shot.

Reply:
Added the partial in https://github.com/hashicorp/vault/pull/15892/commits/415015cb1b4d2166300a534c4696c6e3265686b3. Thanks!